app/messages.ts

```diff
@@ -98,6 +98,74 @@ export const messages = {
       },
     },
   },
+  householder: {
+    householderIdentityForm: {
+      validation: {
+        nameMinLength: 'نام سرپرست باید حداقل ۳ حرف باشد',
+        fatherNameMinLength: 'نام پدر باید حداقل ۳ حرف باشد',
+        nationalIdMinLength: 'شماره کارت ملی باید حداقل ۱۰ عدد باشد',
+        inValidNationalId: 'لطفا یک شماره کارت ملی معتبر وارد نمایید',
+        inValidSsn: 'لطفا یک شماره شناسنامه معتبر وارد نمایید',
+        ssnMinLength: 'شماره کارت ملی باید حداقل ۸ عدد باشد',
+      },
+      title: 'اطلاعات هویت',
+      submitBtn: 'ثبت',
+      nameInput: {
+        label: 'نام',
+        placeholder: 'برای مثال: محمد',
+      },
+      lastNameInput: {
+        label: 'نام خانوادگی',
+        placeholder: 'برای مثال: علیان',
+      },
+      fatherNameInput: {
+        label: 'نام پدر',
+        placeholder: 'برای مثال: محمد',
+      },
+      nationalityInput: {
+        data: { value: 'IR', label: 'ایرانی' },
+        label: 'ملیت',
+        placeholder: 'برای مثال: ایرانی',
+      },
+      nationalIdInput: {
+        label: 'شماره کارت ملی',
+        placeholder: 'برای مثال: ۰۱۲۳۴۵۶۷۸۹',
+      },
+      ssnInput: {
+        label: 'شماره شناسنامه',
+        placeholder: 'برای مثال: 12345678',
+      },
+      issuedAtInput: {
+        data: { value: 'tehran', label: 'تهران' },
+        label: 'محل صدور',
+        placeholder: 'برای مثال: تهران',
+      },
+      religionInput: {
+        data: { value: 'islam', label: 'اسلام' },
+        label: 'دین',
+        placeholder: 'برای مثال: اسلام',
+      },
+      genderInput: {
+        data: {
+          value: 'male',
+          label: 'مرد',
+          secondValue: 'female',
+          secondLabel: 'زن',
+        },
```

Comment:
array is a better choice I think
```suggestion
        options: [
          { value: 'male', label: 'مرد' },
          { value: 'female', label: 'زن' },
        ],
```